### PR TITLE
Add oracleLastUpdatedSlot to Bank and PerpMarket

### PR DESCRIPTION
**Pull request by riordanp**
Parses the best value we have for last updated slot and sets it on the Bank and PerpMarkets when reloading oracle prices. Used for checking oracle staleness on the client side.

### CODE REVIEW

File: ts/client/src/accounts/oracle.ts [hunk 1]
```diff
@@ -48,21 +48,29 @@ export class StubOracle {
...
   accountInfo: AccountInfo<Buffer>,
-): number {
-  const aggregatorAccountData =
-    program.decodeLatestAggregatorValue(accountInfo);
-  if (!aggregatorAccountData)
+): { price: number; lastUpdatedSlot: number } {
+  const price = program.decodeLatestAggregatorValue(accountInfo)!.toNumber();
+  const lastUpdatedSlot = program
+    .decodeAggregator(accountInfo)
+    .previousConfirmedRoundSlot!.toNumber();
```
Comment (microwavedcola1) [inline ts/client/src/accounts/oracle.ts:69]: hmm should this rather be latest_confirmed_round? @ckamm

Comment (riordanp) [reply ts/client/src/accounts/oracle.ts]: yes this matches the check in the program, updated

File: package.json [hunk 2]
```diff
@@ -58,6 +57,7 @@
     "trailingComma": "all"
   },
   "dependencies": {
+    "@jup-ag/core": "^1.0.0-beta.28",
```
Comment (tylersssss) [inline package.json:60]: Is this package needed? I don't see it imported anywhere in this PR or in my local copy. 

I think it was added a long time ago and can now be deleted.

Comment (tylersssss) [inline package.json:60]: Is this package needed? I don't see it imported anywhere in this PR or in my local copy. 

I think it was added a long time ago and can now be deleted.

**Review by tylersssss — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (tylersssss) [reply package.json:60]: Not sure how I ended up with 2 of the same comment. Resolving this one.

Comment (riordanp) [reply package.json]: Seems to build ok without it

**Review by microwavedcola1 — APPROVED** [review]

**Review by tylersssss — APPROVED** [review]

**Pull request merged**